api/src/zero/module/xx.clj

```diff
@@ -88,9 +100,18 @@
      :cram_ref_fasta       fasta
      :cram_ref_fasta_index (str fasta ".fai")}))
 
+(defn apply-overrides
+  "Return an map like DEFAULTS but overridden with any RAW-ARGS that have FLAG-PREFIX."
+  [defaults flag-prefix raw-args]
+  (merge defaults (->> raw-args
+                       (filter #(str/starts-with? % (str flag-prefix ":")))
+                       (map #(str/split (subs % 3) #"="))
+                       (map (fn [[k v]] [(keyword k) v]))
+                       (into {}))))
```

Comment:
I can't really tell unreadable clojure from readable clojure, happy to do this some other way if this is confusing for folks